tutorials/shading/shading_reference/spatial_shader.rst

```diff
@@ -272,7 +272,17 @@ To write a light function, assign something to DIFFUSE_LIGHT or SPECULAR_LIGHT.
 means no light is processed.
 
 The light function is called for every light in every pixel. It is called within a loop for
-each light type. 
+each light type.
+
+Light shader example:
```

Comment:
This is fine, but in the rest of the tutorial examples are introduced using a sentence. So, something like "Below is an example of a custom light function using a Lambertian lighting model" might be better.